CMakeLists.txt

```diff
@@ -224,33 +224,33 @@ set(pydynd_CPP_SRC
     )
 set(pydynd_CYTHON_SRC
     dynd/cuda.pyx
-    dynd/dynd._pydynd.pyx
-    dynd/array.pxd
+    dynd/_pydynd.pyx
```

Comment:
The dynd._pydynd name was to make it possible to pickle the types, see https://github.com/libdynd/dynd-python/issues/96. Did you figure out an alternative way to allow this?